app/src/ui/app.tsx

```diff
@@ -1391,9 +1391,12 @@ export class App extends React.Component<IAppProps, IAppState> {
     if (repository) {
       icon = iconForRepository(repository)
       title = repository.name
-    } else {
+    } else if (this.state.repositories.length > 0) {
       icon = OcticonSymbol.repo
       title = __DARWIN__ ? 'Select a Repository' : 'Select a repository'
+    } else {
+      icon = OcticonSymbol.question
```

Comment:
Could we stick to the `OcticonSymbol.repo` icon here please? That should be the base icon since that's the core responsibility of the button.

Reply:
I had a feeling that you will want the icon to stay the same but I "followed" what's proposed in the issue. Doing this change right now.